test/support/helpers/scenarios.js

```diff
@@ -83,20 +79,34 @@ if (module.exports.scenarios.length === 0) {
   )
 }
 
+const disabledScenario = scenario =>
+  typeof scenario.disabled === 'string' && scenario.disabled
+
+/**
+ * For historical reasons, `disabledScenarioTest()` may receive test names
+ * with file extension. Although it may be useless in many cases, stripping
+ * them will make sure code always works.
+ */
+const disabledScenarioTest = (scenario, testName) =>
+  disabledScenario(scenario) ||
+  _.get(scenario, ['disabled', testName.replace(path.extname(testName), '')])
```

Comment:
Why the extra pair of quotes at the end?

Reply:
Because otherwise, `undefined` would be coerced to a string, ending up looking for `atom/linuxundefined` for example?